doc/nrf/gs_assistant.rst

```diff
@@ -54,18 +54,15 @@ Once you have installed the Toolchain manager, open it in nRF Connect for Deskto
 
 Click :guilabel:`Settings` in the navigation bar to specify where you want to install the |NCS|.
 Then, in :guilabel:`SDK Environments`, click the :guilabel:`Install` button next to the |NCS| version that you want to install.
-
 The |NCS| version of your choice is installed on your machine, and the :guilabel:`Install` button changes to :guilabel:`Open IDE`.
-This button lets you start SEGGER Embedded Studio if you want to start :ref:`gs_programming_ses`.
 
-.. figure:: images/gs-assistant_tm_installed.png
-   :alt: The Toolchain manager options after installing the nRF Connect SDK version, cropped
+There are two ways you can build an application:
 
-   The Toolchain Manager options after installing the |NCS| version
+1. To :ref:`build with SES <gs_programming_ses>`, click on the :guilabel:`Open IDE` button.
+#. To :ref:`build on the command line <gs_programming_cmd>`, click on the dropdown menu and select the :guilabel:`Open bash` option.
 
-Additionally, the dropdown menu becomes available.
-Among other options, this menu lets you open bash and command prompt that use the installed Toolchain manager tools.
-You can use the :guilabel:`Open bash` option from this menu when you start :ref:`gs_programming_cmd`.
+   .. note::
+      Complete the instructions in :ref:`build_environment_cli` before :ref:`gs_programming_cmd`.
```

Comment:
Would it be possible to inline the relevant parts instead of linking? I find it hard to go back and forth between pages in the documentation. The section in _Installing manually_ that is linked to also includes several tasks that are not relevant when using the _Toolchain Manager_.

I tried on a fresh computer, and the only thing missing are the [nRF Command Line Tools](https://www.nordicsemi.com/Products/Development-tools/nrf-command-line-tools/download). After installing it's necessary to restart both ​_bash_ and the _Toolchain Manager_ to get nrfjprog in PATH